src/views/virtualmachines/details/tabs/details/VirtualMachineDetailsPage.tsx

```diff
@@ -18,6 +18,14 @@ type VirtualMachineDetailsPageProps = RouteComponentProps<{
   obj?: V1VirtualMachine;
 };
 
+const pathsToHighlight = [
```

Comment:
can ALL pathsToHighlights be saved in a central location? so paths can be reused and shared? i think this approve will be better